src/Octoshift/AdoApi.cs

```diff
@@ -85,7 +85,7 @@ public virtual async Task<string> GetGithubAppId(string org, string githubOrg, I
                 var url = $"https://dev.azure.com/{org}/{adoTeamProject}/_apis/serviceendpoint/endpoints?api-version=6.0-preview.4";
                 var response = await _client.GetWithPagingAsync(url);
 
-                var endpoint = response.FirstOrDefault(x => (string)x["type"] == "GitHub" && ((string)x["name"]).ToLower() == githubOrg.ToLower());
+                var endpoint = response.FirstOrDefault(x => ((string)x["type"]).Equals("GitHub", StringComparison.OrdinalIgnoreCase) && ((string)x["name"]).Equals(githubOrg, StringComparison.OrdinalIgnoreCase));
```

Comment:
Is this related to the ado-team-project change, or is this just a bit of unrelated cleanup?

Reply:
I created #242 because of a warning that is thrown at the end of the process where type returned is all lower case for me so was not matching GitHub